test/integration-new/ipv6/ipv6_host_networking_test.go

```diff
@@ -0,0 +1,280 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package ipv6
```

Comment:
Maybe as a follow up can we also add test cases around the new env variables? ENABLE IPV4 and IPV6?

Reply:
They are covered by UT though. In general we can't toggle these 2 env variables on a live cluster as the cluster can either be an IPv4 or an IPv6 cluster.